src/plugins/home/public/application/components/add_data/add_data.tsx

```diff
@@ -17,69 +17,78 @@
-  <section className="homDataAdd" aria-labelledby="homDataAdd__title">
-    <EuiFlexGroup alignItems="center">
-      <EuiFlexItem grow={1}>
-        <EuiTitle size="s">
-          <h2 id="homDataAdd__title">
-            <FormattedMessage id="home.addData.sectionTitle" defaultMessage="Ingest your data" />
-          </h2>
-        </EuiTitle>
-      </EuiFlexItem>
+export const AddData: FC<Props> = ({ addBasePath, features }) => {
+  const { trackUiMetric } = getServices();
 
-      <EuiFlexItem className="homDataAdd__actions" grow={false}>
-        <div>
-          <EuiButtonEmpty
-            className="homDataAdd__actionButton"
-            flush="left"
-            href="#/tutorial_directory/sampleData"
-            iconType="visTable"
-            size="xs"
-          >
-            <FormattedMessage
-              id="home.addData.sampleDataButtonLabel"
-              defaultMessage="Try our sample data"
-            />
-          </EuiButtonEmpty>
-        </div>
-      </EuiFlexItem>
-    </EuiFlexGroup>
-
-    <EuiSpacer size="m" />
+  return (
+    <section className="homDataAdd" aria-labelledby="homDataAdd__title">
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem grow={1}>
+          <EuiTitle size="s">
+            <h2 id="homDataAdd__title">
+              <FormattedMessage id="home.addData.sectionTitle" defaultMessage="Ingest your data" />
+            </h2>
+          </EuiTitle>
+        </EuiFlexItem>
 
-    <EuiFlexGroup className="homDataAdd__content">
-      {features.map((feature) => (
-        <EuiFlexItem key={feature.id}>
-          <Synopsis
-            id={feature.id}
-            onClick={createAppNavigationHandler(feature.path)}
-            description={feature.description}
-            iconType={feature.icon}
-            title={feature.title}
-            url={addBasePath(feature.path)}
-            wrapInPanel
-          />
+        <EuiFlexItem className="homDataAdd__actions" grow={false}>
+          <div>
+            <EuiButtonEmpty
+              className="homDataAdd__actionButton"
+              flush="left"
+              href="#/tutorial_directory/sampleData"
+              iconType="visTable"
+              size="xs"
+            >
+              <FormattedMessage
+                id="home.addData.sampleDataButtonLabel"
+                defaultMessage="Try our sample data"
+              />
+            </EuiButtonEmpty>
+          </div>
         </EuiFlexItem>
-      ))}
-    </EuiFlexGroup>
-  </section>
-);
+      </EuiFlexGroup>
+
+      <EuiSpacer size="m" />
+
+      <EuiFlexGroup className="homDataAdd__content">
+        {features.map((feature) => (
+          <EuiFlexItem key={feature.id}>
+            <Synopsis
+              id={feature.id}
+              onClick={(event: MouseEvent) => {
+                createAppNavigationHandler(feature.path)(event);
+                trackUiMetric(METRIC_TYPE.CLICK, `ingest_data_card_${feature.id}`);
```

Comment:
I wonder if we want to swap these two lines?

`createAppNavigationHandler` bails if someone goes to open the link in a new tab but we could still collect that metric.